langlib/lang.xml/src/main/java/org/ballerinalang/langlib/xml/utils/Lists.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://wso2.com).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ballerinalang.langlib.xml.utils;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * List util class for the langlib xml module.
+ *
+ * @since 2201.9.0
+ */
+public class Lists {
```

Comment:
Shall we rename the class name and the method name to be more meaningful? 
something like `XmlUtils.getList()`?